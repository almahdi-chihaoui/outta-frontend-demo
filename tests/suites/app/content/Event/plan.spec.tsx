import React from "react";
import renderer from "react-test-renderer";

import Plan from "../../../../../src/containers/app/content/Event/plan";

describe('Plan Component', () => {
  let component;

  beforeEach(() => {
    component = renderer.create(
      <Plan></Plan>
    );
  });

  it('should render', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});