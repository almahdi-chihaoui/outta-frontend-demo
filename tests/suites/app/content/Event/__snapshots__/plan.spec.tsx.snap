// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plan Component should render 1`] = `
<div
  className="makeStyles-root-2"
>
  <div
    className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation1"
  />
  <div
    className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation1"
  />
  <div
    className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation1"
  />
  <div
    className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation1"
  />
</div>
`;
